assets/js/components/ClusterDetails/ChecksResults.jsx

```diff
@@ -69,6 +73,14 @@ export const ChecksResults = () => {
     ]
   );
 
+  const checksRemediationById = Object.assign(
```

Comment:
This approach works, but I find more "normal" to run a `find` function:
```
export getRemediationById = (id) => (catalogData) =>
  catalogData.flat().find((check) => check.id === id);
```
(didn't test, but I guess you get the idea)